203 new icons

baitmooth opened the pull request:
-- | -- | -- | -- | --
-- | -- | -- | -- | --
activebuilding (*n*)| activitywatch (*n*)| adm (*n*)| adobefillsign (*n*)| agendalite (*n*)
airtable (*n*)| airywaretuner (*n*)| ameixamonochrome (*n*)| anchor (*n*)| andronix (*n*)
anonaddy (*n*)| antennapod (*u*)| anutotd (*n*)| appreactornews (*n*)| autoairplanemode (*n*)
aves (*n*)| azureauthenticator (*n*)| b4ufly (*n*)| balancetheball (*n*)| banking4 (*n*)
banking (*n*)| barinsta (*n*)| bestbuy (*n*)| binance (*n*)| bing (*n*)
bluetoothkeyboardmouse (*n*)| boursoramabanque (*n*)| budgetmylife (*n*)| calyxosrecorder (*n*)| campfire (*n*)
carreport (*n*)| circuittrainer (*u*)| clima (*n*)| contactscircle (*n*)| contactssync (*n*)
convergexperience (*n*)| cuprumpdf (*u*)| dal (*n*)| darq (*n*)| dawn (*n*)
deepl (*n*)| deliveries (*n*)| desmos (*n*)| detoxdroid (*n*)| dhlexpress (*n*)
dhlpaket (*n*)| diaguard (*n*)| dial (*n*)| discourse (*n*)| diskusage (*u*)
ditmm (*n*)| dlive (*n*)| documents (*n*)| drastic (*u*)| droidcam (*n*)
dronecast (*n*)| dropticket (*n*)| elegantteleprompter (*n*)| elementary (*n*)| emeralddialer (*n*)
enki (*n*)| espeak (*n*)| etchdroid (*n*)| exactcalculator (*n*)| exodus (*n*)
familylink (*n*)| fdroidclassic (*n*)| fdroidforum (*n*)| filmstaden (*u*)| financemanager (*u*)
firewall (*n*)| fissh (*n*)| fitbitcoach (*n*)| flipaclip (*n*)| flow (*n*)
flstudiomobile (*n*)| foxitpdf (*n*)| foxydroid (*u*)| freifunkkarte (*n*)| geburtstagskalender (*n*)
geohashdroid (*n*)| geologischezeitskala (*n*)| georinexlogger (*n*)| gitter (*n*)| gittouch (*n*)
glxy (*n*)| go (*n*)| groupme (*n*)| gtasks (*n*)| guardian (*n*)
haven (*n*)| headi (*n*)| heartbeat (*n*)| helm (*u*)| helmsync (*u*)
imagevideodatefixer (*n*)| inaturalist (*n*)| informeddelivery (*n*)| iprintscan (*n*)| iwanttfc (*n*)
jellyfin (*u*)| justeat (*n*)| keepa (*n*)| keepon (*n*)| keybase (*n*)
kickstarter (*n*)| kinderundgift (*n*)| kinemaster (*n*)| kleinewettervorschaudeutschland (*n*)| kobocollect (*n*)
lbry (*n*)| linguee (*n*)| linksys (*n*)| logischeselbstverteidigung (*n*)| mastermindy (*n*)
math (*n*)| mediasetplay (*n*)| meinmagenta (*n*)| merlin (*n*)| meshtastic (*n*)
minesweeper (*u*)| mistore (*n*)| motioneyeapp (*n*)| moviedb (*u*)| msnews (*n*)
multivnc (*n*)| myboy (*u*)| mycanal (*n*)| myfiles (*u*)| myfritz (*n*)
mytim (*n*)| napster (*u*)| navigator (*n*)| ncalcplus (*n*)| ncpasswords (*n*)
neolms (*n*)| nextcloudnotes (*u*)| nextcloudsms (*u*)| nextcloud (*u*)| nova7 (*n*)
nprnews (*n*)| nprone (*n*)| nzb360 (*n*)| octoremote (*n*)| odkcollect (*n*)
omletarcade (*n*)| openbeautyfacts (*n*)| opencvmanager (*n*)| openpetfoodfacts (*n*)| organicmaps (*n*)
otomusic (*n*)| phonedial (*n*)| photoeditor (*n*)| pianopartner2 (*n*)| piholedroid (*n*)
piwigo (*n*)| prayertimes (*n*)| prepaidguthaben (*n*)| proexpense (*n*)| qfield (*n*)
qgroundcontrol (*n*)| qr (*u*)| quickbible (*n*)| quicksupport (*n*)| quranformyandroid (*n*)
raspberry (*u*)| raspcontroller (*n*)| react (*n*)| reckoningskills (*n*)| rinexon (*n*)
rocksndiamonds (*n*)| rokuremote (*n*)| roommate (*n*)| ruler (*u*)| samsunghealth (*n*)
satorbit (*n*)| securego (*n*)| secusoshoppinglist (*u*)| shadereditor (*n*)| showly20 (*n*)
sknetrafiken (*u*)| slimsocial (*n*)| snippet (*n*)| sofascore (*n*)| solex (*n*)
somafm (*n*)| soundhound (*n*)| stockswidget (*n*)| stormradar (*n*)| streaming (*n*)
studiumunict (*n*)| sudoku (*u*)| supertuxkart (*n*)| sygic (*n*)| tan2go (*n*)
tapemeasure (*n*)| tasker (*u*)| teamviewer (*n*)| termius (*n*)| terragenesis (*n*)
tether (*n*)| tibackup (*u*)| tiktok (*n*)| todo (*n*)| transmissionremote (*n*)
trebleinfo (*n*)| unocalc (*n*)| ups (*n*)| uspsmobile (*n*)| vancedmanager (*n*)
video (*n*)| videotranscoder (*n*)| vigilante (*n*)| vncviewer (*n*)| vrmediaplayer (*n*)
watomatic (*n*)| weatherwidget (*n*)| websitemonitor (*n*)| webvideocaster (*n*)| whatsdeleted (*n*)
wolframalpha (*n*)| wookiee (*n*)| wordreference (*n*)| wrongpinshutdown (*n*)| xbox (*n*)
xdadevelopers (*n*)| yaga (*n*)| yoroi (*n*)| ytmusic (*n*)| yuzubrowser (*n*)
zapp (*u*)|
*total icons*: **231** | **203** new icons (*n*)| **28** updated icons (*u*)

![main-2021-10-03-1822](https://user-images.githubusercontent.com/23211478/135763219-40b3865d-168d-4688-bbb9-1718a98edc29.png)

baitmooth committed:
Merge branch 'master' of https://github.com/dkanada/frost into icons

baitmooth committed:
resized icons

and reworked antennapod

baitmooth committed:
5 new icons

elementary
foxydroid (unfinished first try)
kleinewettervorschau
shadereditor (also unfinished)
slimsocial (looks to much like a contacts icon, needs review)

baitmooth committed:
2 new icons

bluetoothkeyboardmouse
multivnc

baitmooth committed:
2 new icons

myfritz
supertuxkart

baitmooth committed:
updated appfilter

added:
- Bluetooth Keyboard & Mouse
- Elementary
- Equate
- Kleine Wettervorschau Deutschland
- MultiVNC
- MyFRITZ
- Shader Editor
- SlimSocial
- SuperTuxKart

baitmooth committed:
updated

bluetoothkeyboardmouse.svg
foxydroid.svg
myfritz.svg
nextcloudsms.svg
supertuxkart.svg

baitmooth committed:
2 new icons

react
yuzubrowser

baitmooth committed:
4 new icons

- anutotd
- carreport
- fdroidclassic
- prayertimes

baitmooth committed:
updated appfilter

baitmooth committed:
removed duplicate entry

anutotd

baitmooth committed:
3 new icons

activebuilding
bestbuy
somafm

baitmooth committed:
3 new icons

- bing
- mastermindy
- weatherwidget

baitmooth committed:
optimized foxydroid

baitmooth committed:
updated carreport

had the png still embedded

baitmooth committed:
5 new icons

fitbitcoach
gitt
qfield
soundhound
vancedmanager

dkanada commented:
Not relevant to your new icons but the FoxyDroid icon has always looked weird if you want to fix it up. The triangles don't line up at all, probably because the three gaps are doing something weird.

baitmooth committed:
4 new icons

darq
meinmagenta
whatsdeleted
wrongpinshutdown

baitmooth committed:
fixed some errors in svg's
antennapod.svg
moviedb.svg
nextcloud.svg
qr.svg

baitmooth commented:
> Not relevant to your new icons but the FoxyDroid icon has always looked weird if you want to fix it up. The triangles don't line up at all, probably because the three gaps are doing something weird.

Yes. I still work on this strange effect. Downloaded the original svg from their repo and now I have a clue on how to fix it. There is a small angle in it which makes the original look good. But I still need to tweak the size of the gaps.

baitmooth committed:
4 new icons

autoairplanemode.svg
dal.svg
geburtstagskalender.svg
imagevideodatefixer.svg

baitmooth committed:
removed duplicate entry in appfilter.xml

Dal

baitmooth commented:
![which](https://user-images.githubusercontent.com/23211478/121697444-5d753180-cacd-11eb-967b-9a56422a0ef2.png)
'autoairplanemode' is some kind of moon with craters I tried to imitate. don't know which is better. If none I will try something different.

baitmooth committed:
4 new icons
banking4.svg
logischeselbstverteidigung.svg
securego.svg
tan2go.svg

baitmooth committed:
4 new icons

new:
binance.svg
contactssync.svg
rokuremote.svg
todo.svg

rename:
kleinewettervorschau.svg -> kleinewettervorschaudeutschland.svg

baitmooth committed:
4 new icons 1 update

new:
adobefillsign.svg
andronix.svg
fissh.svg
gittouch.svg

update:
helmsync.svg

baitmooth committed:
4 new icons

azureauthenticator.svg
groupme.svg
msnews.svg
news.svg

baitmooth committed:
renamed news -> appreactornews

baitmooth committed:
updated icons

helm.svg
helmsync.svg
tibackup.svg
weatherwidget.svg

baitmooth committed:
8 new icons

barinsta.svg
elegantteleprompter.svg
exodus.svg
keepa.svg
opencvmanager.svg
piholedroid.svg
sofascore.svg
transmissionremote.svg

baitmooth committed:
3 new icons & 2 fixed

documents.svg
etchdroid.svg
motioneye.svg

baitmooth committed:
10 new icons 1 update
agendalite.svg
anonaddy.svg
flow.svg
haven.svg
inaturalist.svg
roommate.svg
vrmediaplayer.svg
watomatic.svg
websitemonitor.svg
yaga.svg

baitmooth committed:
ungrouped elements in yaga.svg

baitmooth committed:
yaga again, missed stripping m)

baitmooth committed:
renamed motioneye -> motioneyeapp

conflicting filenames which i didn't check earlier. seem to be different
apps.

baitmooth committed:
10 new icons

desmos.svg
droidcam.svg
dropticket.svg
headi.svg
keepon.svg
mistore.svg
mytim.svg
ncpasswords.svg
piwigo.svg
proexpense.svg

baitmooth committed:
10 new icons

clima.svg
dawn.svg
detoxdroid.svg
familylink.svg
kickstarter.svg
kobocollect.svg
otomusic.svg
raspcontroller.svg
rinexon.svg
studiumunict.svg

baitmooth committed:
3 new, 6 updates

new:
lbry.svg
tapemeasure.svg
webvideocaster.svg

added secuso logo:
circuittrainer.svg
financemanager.svg
minesweeper.svg
ruler.svg
secusoshoppinglist.svg
sudoku.svg

baitmooth committed:
stripped 3 icons

missed stripping and ungrouping

baitmooth committed:
added detail to tapemeasure

baitmooth committed:
appfilter.xml for #480

baitmooth committed:
6 new icons

quranformyandroid.svg
reckoningskills.svg
satorbit.svg
solex.svg
sygic.svg
wookiee.svg

baitmooth committed:
2 new icons 1 update

emeralddialer.svg
openbeautyfacts.svg

raspberry.svg - removed common shape

baitmooth commented:
If this gets merged, #311 #328 #336 #351 #369 #389 #390 #391 #392 #399 #400 #413 #431 #436 #474 #480 can be closed.

I did not add anything to it but #119 #305 #331 #332 #377 #384 #395 are also done.

baitmooth committed:
7 new icons

dronecast.svg
geohashdroid.svg
linksys.svg
photoeditor.svg
quicksupport.svg
teamviewer.svg
wolframalpha.svg

baitmooth committed:
foxydroid

tried to fix the gaps

baitmooth committed:
1 new icon

iwanttfc.svg

baitmooth commented:
https://github.com/dkanada/frost/pull/475/commits/a1803687c96bfc0b1d154c6dc73cfc6c57ea0607

![foxydroid](https://user-images.githubusercontent.com/23211478/123632158-e7d0cb80-d817-11eb-849a-c361ac0fea69.png)

(earlier preview was only visible in gh darkmode)
I would call this my final attempt to fix foxydroids strange effect on the gaps. It's very close to the original regarding angles and positions, plus gaps.
To me it looks much better then my previous tries.

baitmooth committed:
ncalcplus

baitmooth committed:
4 new icons 3 changed

new:
airywaretuner.svg
enki.svg
streaming.svg
xbox.svg

changed:
foxydroid.svg
linksys.svg
photoeditor.svg

appfilter.xml

baitmooth committed:
4 new icons

adm.svg
convergexperience.svg
dlive.svg
flstudiomobile.svg

baitmooth committed:
7 new icons

anchor.svg
foxitpdf.svg
iprintscan.svg
justeat.svg
keybase.svg
kinemaster.svg
omletarcade.svg

baitmooth committed:
8 new icons

dhlexpress.svg
math.svg
merlin.svg
nprone.svg
qgroundcontrol.svg
terragenesis.svg
ups.svg
videotranscoder.svg

baitmooth committed:
7 new icons

gtasks.svg
mediasetplay.svg
nova7.svg
tether.svg
vigilante.svg
wordreference.svg
xdadevelopers.svg

baitmooth commented:
The icon nova7 is for a launcher app with activity name already covered by another icon. But the app name differs.

baitmooth committed:
2 new icons & 'Simple Dailer' #486

Credits for these go to @frosticons

linguee.svg
ytmusic.svg

baitmooth committed:
sorted appfilter.xml

baitmooth commented on app/src/main/res/xml/appfilter.xml:
```diff
@@ -2310,10 +2559,16 @@
     <!-- Notizen -->
     <item component="ComponentInfo{it.niedermann.owncloud.notes/it.niedermann.owncloud.notes.android.activity.SplashscreenActivity}" drawable="nextcloudnotes" />
 
+    <!-- Nova7 -->
```
not sure about this one.

dkanada replied:
What app does it conflict with?

baitmooth replied:
I think the one below. 
'Nova Launcher'.

baitmooth committed:
3 icons for #400

dhlpaket.svg
diaguard.svg
yoroi.svg

also added new linphone activity to appfilter.xml

baitmooth committed:
samsunghealth.svg for #400

baitmooth committed:
2 new icons for 437

baitmooth committed:
2 new icons for #437

boursoramabanque.svg
mycanal.svg

baitmooth committed:
Merge branch 'icons' of github.com:baitmooth/frost into icons

baitmooth committed:
icons for #488

baitmooth committed:
dial

was drawn by @frosticons but is now without an app

baitmooth committed:
minimal changes to some icons

- budgetmylife -> made $-sign look more centerd
-firewall -> resized inner rectangle and symbol
- kinderundgift -> removed unneeded nodes in, and slightly reshaped
scull
- rest is because of scour rerun

baitmooth committed:
6 icons for #412

deliveries.svg
meshtastic.svg
nzb360.svg
octoremote.svg
stormradar.svg
trebleinfo.svg

baitmooth committed:
1 icon for #412

pianopartner2.svg

baitmooth committed:
5 new icons for #490

changed:       app/src/main/res/xml/appfilter.xml

new:
aves.svg
deepl.svg
quickbible.svg
showly20.svg
stockswidget.svg

baitmooth committed:
updated appfilter.xml

- removed duplicate entry in Calculator
- added entry for 'Pdf Viewer Plus'

baitmooth committed:
5 new icons for #393

baitmooth committed:
changed files

changed:
- boursoramabanque.svg
- flipaclip.svg
- iprintscan.svg
- keybase.svg
- math.svg
- merlin.svg
- stockswidget.svg

baitmooth committed:
1 new icon for #492

changed:
- appfilter.xml

new:
- snippet.svg

baitmooth commented:
Since my harddisk failed and i can't replace it soon there will probably be no update from me here in the near future. 
I was so close to 200 (ノ°益°)ノ

dkanada commented:
No worries, this is still probably the largest pull request I've ever received 😆 I'm extremely busy this month but I will review it some time early next month.

baitmooth commented:
Turns out the failing harddrive was broken RAM. So I'm back to do some more icons. 
╰(▔∀▔)╯

baitmooth committed:
5 new icons for #327

changed:
app/src/main/res/xml/appfilter.xml

new:
b4ufly.svg
go.svg
guardian.svg
navigator.svg
odkcollect.svg

baitmooth committed:
6 new icons for #327

changed:
app/src/main/res/xml/appfilter.xml

new:
discourse.svg
georinexlogger.svg
informeddelivery.svg
nprnews.svg
unocalc.svg
uspsmobile.svg

baitmooth committed:
2 new icons 5 updated

new:
- openpetfoodfacts
- prepaidguthaben

changed:
- appfilter.xml
- airtable:
    right rectangle looked to big
- airywaretuner:
    better sine wave, some resizing
- dhlpaket:
    redrawn box
- satorbit:
    uniform shape of radiation
- go
    changed by scour

baitmooth committed:
6 new icons #431

changed:
app/src/main/res/xml/appfilter.xml

new:
activitywatch.svg
contactscircle.svg
exactcalculator.svg
phonedial.svg
tiktok.svg
video.svg

baitmooth committed:
banking for #490

baitmooth committed:
updated and new icons
